src/classes/groundplaces.ts

```diff
@@ -25,42 +34,54 @@ export class GroundPlaces {
   public autocomplete(query: string, filters: AutoCompleteFilters[]): AutoComplete {}
```

Comment:
```
@description Returns a list of places.
```

This output shouldn't be an array like that: 

```
Autocomplete[]
```

What do you think @Theo-Le-Donne @Simobain ?

Reply:
Yes an array for output !